lsp-ui-sideline.el

```diff
@@ -426,26 +429,26 @@ Push sideline overlays on `lsp-ui-sideline--ovs'."
       (let* ((lines (--> (flycheck-error-format-message-and-id e)
                       (split-string it "\n")
                       (lsp-ui-sideline--split-long-lines it)))
-             (display-lines (butlast lines (- (length lines) lsp-ui-sideline-diagnostic-max-lines)))
+             (display-lines (nreverse (butlast lines (- (length lines) lsp-ui-sideline-diagnostic-max-lines))))
              (offset 1))
-        (dolist (line (nreverse display-lines))
-          (let* ((message (string-trim (replace-regexp-in-string "[\t ]+" " " line)))
-                 (len (length message))
+        (dolist (line display-lines)
+          (let* ((msg (string-trim (replace-regexp-in-string "[\t ]+" " " line)))
+                 (msg (replace-regexp-in-string " " " " msg))
```

Comment:
This is the same as 

https://github.com/emacs-lsp/lsp-ui/blob/c5b094d330c7dec3ebcdbb1e129b53f7c7b89587/lsp-ui-doc.el#L916